pkg/liquidity-mining/contracts/gauges/ChildChainGauge.vy

```diff
@@ -110,17 +110,16 @@ reward_integral_for: public(HashMap[address, HashMap[address, uint256]])
 # user -> token -> [uint128 claimable amount][uint128 claimed amount]
 claim_data: HashMap[address, HashMap[address, uint256]]
 
-is_killed: public(bool)
```

Comment:
I'd like to give this one last thought.

If it's really unnecessary then these changes are OK, but are we sure we don't want to implement a `kill` functionality?
I guess you can always do that with the root gauges though right?

Reply:
Right; the point is that if we don't have the kill behavior in L2s, the only thing that we'd be missing is getting the already bridged BAL out of circulation. Besides that, it's not that it continues to be a problem after the L1 counterpart is killed.

If you think it's still worth it, let's go for it.

Reply:
It seems it's a bit more simple than in the liquidity gauge, since in this case the loop just accumulates inflation in a variable and we can just skip it if killed.
See https://github.com/balancer-labs/balancer-v2-monorepo/pull/2306/commits/cf0dac798ab58c53d615d3cde2f7bffd4f7cf4d0.